tests/integration/feature_store_integration/test_feature_materialize.py

```diff
@@ -626,12 +626,35 @@ async def test_feature_tables_populated(session, offline_store_feature_tables_al
             assert (df[DUMMY_ENTITY_COLUMN_NAME] == DUMMY_ENTITY_VALUE).all()
 
 
+async def _check_udf_with_container_input(
+    session, udfs_for_on_demand_func, offline_store_feature_tables
+):
+    """Test that udf is created in databricks"""
+    # check UDF taking count dictionary input as input
+    table_name = "cat1_cust_id_1h"
+    cos_sim_udf = next(
+        udf
+        for udf in udfs_for_on_demand_func
+        if udf.startswith("udf_external_fs_cosine_similarity_")
+    )
+    schema_name = session.schema_name
+    table = offline_store_feature_tables[table_name]
+    assert len(table.output_column_names) == 1
+    col_name = table.output_column_names[0]
```

Comment:
Is this part easy to break if we add more features to the integration test? If so would be nice if it can be more robust to that, e.g. selecting the desired column explicitly.